core/python2Action/pythonrunner.py

```diff
@@ -0,0 +1,87 @@
+"""Executable Python script for running Python actions.
```

Comment:
Docker does not allow you to get local resources that are not in the current context (i.e. the build dir).  https://travis-ci.org/openwhisk/openwhisk/builds/213370715#L3568  Reduplicating pythonrunner.py